examples/samplers/DEMetropolisZ_EfficiencyComparison.ipynb

```diff
@@ -4,15 +4,13 @@
    "cell_type": "markdown",
```

Comment:
<p>Is that just me, or is NUTS the <em>least</em> reproducible here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>I see what you are looking at.  <code>NUTs</code> does have lower <code>ESS_tail</code> which could explain the additional variability compared to <code>DEMetropolis</code>   Incidentally, more variation in the MCMC results would make is easier to "pass" the t-test (have a high p-value).  I'm suggesting we do not change anything for this exploratory notebook, but it does raise the question on how one might systematically test samplers for accuracy.     </p><p> </p>